app/src/models/metadata.model.js

```diff
@@ -27,4 +30,25 @@ const Metadata = new Schema({
     status: { type: String, enum: STATUS, default: 'published' }
 });
 
+Metadata.index(
```

Comment:
So Mongoose is smart enough to use this and create the index automatically. yay

BUUUTTTT it's not smart enough to update existing indexes. I think we already have a different version of this index in prod, so we might run into trouble when deploying. The fix is dropping the current index, and deploying. Mongoose will take it from there.